src/picknik_ur_gazebo_config/description/picknik_ur_gazebo.xacro

```diff
@@ -125,8 +125,7 @@
       name="ign_ros2_control::IgnitionROS2ControlPlugin"
     >
       <parameters>
-        $(find
-        picknik_ur_gazebo_config)/config/control/picknik_ur5e.ros2_control.yaml
+        $(find picknik_ur_gazebo_config)/config/control/picknik_ur5e.ros2_control.yaml
```

Comment:
Maybe we can get the prettier to skip over this line using ignore comments?
```
<!-- prettier-ignore-start -->
$(find picknik_ur_gazebo_config)/config/control/picknik_ur5e.ros2_control.yaml
<!-- prettier-ignore-end -->
```

https://github.com/prettier/plugin-xml?tab=readme-ov-file#ignore-ranges